src/api/app/models/kiwi/image.rb

```diff
@@ -48,43 +49,51 @@ def self.build_from_xml(xml_string, md5)
     order = 1
     repositories = xml["repository"]
     repositories = [xml["repository"]] if xml["repository"].is_a?(Hash)
-    repositories.each do |repository|
-      attributes = {
-        repo_type:   repository['type'],
-        source_path: repository['source']['path'],
-        priority:    repository['priority'],
-        order:       order,
-        alias:       repository['alias'],
-        replaceable: repository['status'] == 'replaceable',
-        username:    repository['username'],
-        password:    repository['password']
-      }
-      attributes['imageinclude'] = repository['imageinclude'] == 'true' if repository.key?('imageinclude')
-      attributes['prefer_license'] = repository['prefer-license'] == 'true' if repository.key?('prefer-license')
-
-      new_image.repositories.build(attributes)
-      order += 1
+
+    if repositories
+      new_image.use_project_repositories = true if repositories.map { |repository| repository['source']['path'] }.include?('obsrepositories:/')
```

Comment:
``
new_image.use_project_repositories = repositories.any? { |repository| repository['source']['path'] == 'obsrepositories:/'
``

Reply:
Btw: Have you tried if ``obsrepositories://`` is also valid?

Reply:
@ChrisBr, `obsrepositories://` is not valid, should it be valid?

Reply:
As @adrianschroeter wrote it shouldn't be valid, everything fine!